blog/2025/clio-2.5.0.md

```diff
@@ -0,0 +1,258 @@
+- Add ability to create releases in CI ([#2168](https://github.com/XRPLF/clio/pull/2168)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add new Docker images structure, tools image ([#2185](https://github.com/XRPLF/clio/pull/2185)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add ubsan build to nightly release ([#2190](https://github.com/XRPLF/clio/pull/2190)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add support for Permissioned DEX ([#2152](https://github.com/XRPLF/clio/pull/2152)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Switch Clio to use Conan v2 ([#2179](https://github.com/XRPLF/clio/pull/2179)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Always build with native arch in Conan 2 ([#2201](https://github.com/XRPLF/clio/pull/2201)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Pass sanitizer as part of `conan_profile` ([#2189](https://github.com/XRPLF/clio/pull/2189)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add flags to deps for sanitizer builds ([#2205](https://github.com/XRPLF/clio/pull/2205)) by [@godexsoft](https://github.com/godexsoft)
+- Add read-write switching in ETLng ([#2199](https://github.com/XRPLF/clio/pull/2199)) by [@godexsoft](https://github.com/godexsoft)
+- Build GCC natively and then merge the image ([#2212](https://github.com/XRPLF/clio/pull/2212)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add Conan lockfile ([#2220](https://github.com/XRPLF/clio/pull/2220)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Build all possible Conan configurations in CI ([#2225](https://github.com/XRPLF/clio/pull/2225)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add start and finish sequence support in ETLng ([#2226](https://github.com/XRPLF/clio/pull/2226)) by [@godexsoft](https://github.com/godexsoft)
+- Add sanitizer support for macOS dependency builds([#2240](https://github.com/XRPLF/clio/pull/2240)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add graceful shutdown and cleanup for ETLng ([#2232](https://github.com/XRPLF/clio/pull/2232)) by [@godexsoft](https://github.com/godexsoft)
+- Build sanitizers with Clang ([#2239](https://github.com/XRPLF/clio/pull/2239)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add parallel download and upload for Conan packages ([#2254](https://github.com/XRPLF/clio/pull/2254)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add `init_conan script` ([#2242](https://github.com/XRPLF/clio/pull/2242)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add support for Token Escrow ([#2252](https://github.com/XRPLF/clio/pull/2252)) by [@mathbunnyru](https://github.com/mathbunnyru)
+- Add support for Single Asset Vault ([#1979](https://github.com/XRPLF/clio/pull/1979)) by [@PeterChen13579](https://github.com/PeterChen13579)
+- Switch to xrpl/2.5.0 release ([#2267](https://github.com/XRPLF/clio/pull/2267)) by [@mathbunnyru](https://github.com/mathbunnyru)
+
+### Bug Fixes
```

Comment:
I've always seen bug fixes written in past tense. I'd probably apply past tense to all sections below, too.